test/rules/assertions/requireReturns.js

```diff
@@ -134,6 +145,33 @@ export default {
       ],
       parserOptions: {
         ecmaVersion: 8
+      },
+      settings: {
+        jsdoc: {
+          forceRequireReturn: true
+        }
+      }
+    },
+    {
+      code: `
+          /**
+           *
+           */
+           async function quux () {}
+      `,
+      errors: [
+        {
+          line: 2,
+          message: 'Missing JSDoc @returns declaration.'
+        }
+      ],
+      parserOptions: {
+        ecmaVersion: 8
+      },
+      settings: {
```

Comment:
This should be options. (and other places.)

Reply:
`forceRequireReturn` wasn't a part of this PR, so those are still there are settings, but my recent commit did add one bad `forceRequiresWithAsync` which I've now amended, thanks!